app/models/concerns/episode_media.rb

```diff
@@ -133,6 +133,19 @@ def media_ids
     media.map(&:id)
   end
 
+  def uncut=(uncut)
+    # if uncut is unchanged or missing, update other attributes
+    if (uncut && self.uncut) && (uncut.href.blank? || (self.uncut.href == uncut.href))
+      self.uncut.segmentation = uncut.segmentation if uncut.segmentation.present?
+    else
+      super
+    end
+
+    self.medium = if uncut.present?
+      "uncut"
+    end
+  end
```

Comment:
re: https://github.com/PRX/feeder.prx.org/pull/1201#discussion_r2010276485

Reply:
that is what I recall (that would break lots of stuff), I can add a test for it if you're concerned?

Reply:
hmm, there was a thing a long time ago I am remembering about patch vs put for this, where fetch could be sparse, but if you had missing attributes on put, they would get nilled out. I'll try it!

Reply:
Must be an internal bit of represantable, right?  Generally I'm ok with assuming your dependencies do their jobs - mostly trying to remember how it works for my own education!

Reply:
it works! I added a check for this in the api/auth/episodes_controller.rb below ...